src/msg.lisp

```diff
@@ -75,9 +75,26 @@
 (defmethod symbol-codes ((msg-type symbol))
   nil)
 
+(defmethod symbol-codes ((m ros-message))
+  (symbol-codes (type-of m)))
+
 (defmethod symbol-code ((m ros-message) s)
   (symbol-code (type-of m) s))
 
+(defmethod code-symbols ((msg-type symbol) code)
+  (remove code (symbol-codes msg-type) :test-not #'= :key #'rest))
+
+(defmethod code-symbols ((m ros-message) code)
+  (code-symbols (type-of m) code))
+
+(defmethod code-symbol ((msg-type symbol) code)
+  (let ((pair (rassoc code (symbol-codes msg-type) :test #'=)))
+    (when pair
```

Comment:
Could we unify the way corner cases are handled here, i.e. between `code-symbol` and `symbol-code`? Either get rid of errors in both or throw errors in both. Otherwise it might get confusing. I like it when errors are thrown in corner cases as it makes debugging much simpler and returning `NIL` is pretty generic and doesn't tell you much about what went wrong. But in this case it can very well be that one symbol or number is not defined in the message and I guess considering that an error is too harsh. However, I just grepped `symbol-code` in our code base and it's used all over the place. The errors are not handled anywhere but I'd guess, if this is such a widely-used function (and maybe not only by CRAM people) it might be too late to change its behaviour...